src/os/bluestore/BlueStore.cc

```diff
@@ -8095,7 +8102,27 @@ BlueStore::OnodeRef BlueStore::fsck_check_objects_shallow(
 
   num_spanning_blobs += o->extent_map.spanning_blob_map.size();
 
-  o->extent_map.fault_range(db, 0, OBJECT_MAX_SIZE);
+  bool fully_loaded = o->extent_map.fault_range(db, 0, OBJECT_MAX_SIZE);
+  if (!fully_loaded) {
+    // object is corrupted
+    derr << "fsck error: " << oid << " failed to load some shards" << dendl;
```

Comment:
I'd prefer log ouput prefixed with "fsck error:" to be in-sync with reported error count. I.e. amount of such lines in the log to be equal to the number of detected error which fsck reports on completion. Ths line violates that condition